tfhe/src/integer/gpu/ciphertext/mod.rs

```diff
@@ -70,6 +71,60 @@ pub trait CudaIntegerRadixCiphertext: Sized {
     fn gpu_indexes(&self) -> &[GpuIndex] {
         &self.as_ref().d_blocks.0.d_vec.gpu_indexes
     }
+
+    fn to_integer_radix_ciphertext_vec(
```

Comment:
could you add a comment explaining what this function does? something like: "converts a single radix ciphertext (e.g. stored as a contiguous array of LWE blocks) into a ... "